hex/spec/fixtures/mixfiles/nerves

```diff
@@ -0,0 +1,67 @@
+defmodule NervesApp.MixProject do
```

Comment:
If you move this file to `hex/spec/fixtures/projects/nerves/mix.exs` the test should pass

Reply:
Thanks - I missed your comment and fixed it a slightly different. I think this is consistent with the other tests in the directory, but I'll certainly change if that way is not preferred.

Reply:
Yeah, we're moving away from this test setup, and prefer to set up projects that mimic an actual repo. But this is fine for now and we'll migrate all these hex specs in one go at some point